binding_generator.py

```diff
@@ -109,6 +109,8 @@ def generate_builtin_bindings(api, output_dir, build_config):
     for builtin_api in api["builtin_classes"]:
         if is_pod_type(builtin_api["name"]):
             continue
+        if is_included_type(builtin_api["name"]):
```

Comment:
I think we should have the same check in `print_file_list`.

Reply:
That makes sense, thanks :)